### PR TITLE
ci: Setup versioning for releases

### PR DESCRIPTION
**Pull request by abatilo-isp**
These changes will let us version the repo so that as we start putting
more and more actions into this repository that will ultimately be
re-used by other repos (like any helmfile or terraform actions), it'll
be great to be able to version these so that we don't run into risk of
breaking other consuming repos.

### CODE REVIEW

**Comment by mdobosz-isp** [comment]
Are the files in the right spot? They're all under `/helmfile-actions/`
Also, since this is a mono-repo with multiple actions we'll probably need to use a version tag pattern like `actionname-vX.Y.Z`. Do you know if semantic release supports that? I can look myself later too.

**Comment by abatilo-isp** [comment]
> Are the files in the right spot? They're all under `/helmfile-actions/`
> Also, since this is a mono-repo with multiple actions we'll probably need to use a version tag pattern like `actionname-vX.Y.Z`. Do you know if semantic release supports that? I can look myself later too.

Whoops, I definitely put those in the wrong directory.

As for versioning individual actions, I don't think it really matters. Think of the tag as a snapshot of the state of the entire repository, not as the version of an individual single action. If you view it from that angle, then it doesn't really matter.

This is _effectively_ how monorepos get "versioned" at other companies that do really large monorepos. You just care about having an immutable representation of the state of the repository for when your application got built.

Let me fix the directories.

File: helmfile-actions/.releaserc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (mdobosz-isp) [inline helmfile-actions/.releaserc:1]: Shouldn't this file move up a level too?

Comment (abatilo-isp) [reply helmfile-actions/.releaserc:1]: 🤦  thank you

**Pull request merged**